common/src/region.rs

```diff
@@ -170,6 +180,55 @@ impl RegionDefinition {
         })
     }
 
+    // Compare two RegionDefinitions and verify they are compatable.
+    // Compatable is valid if all fields are the same, expect for the
+    // UUID. The UUID should be different.
```

Comment:
Out of curiosity, why can't the UUIDs match?  It seems like a region should be (tautologically) compatible with itself.

Reply:
From a strict comparison point of view, yes it would be compatable.

In this case, if you were trying to clone a downstairs, then your new downstairs should not
have the same UUID as the original.  So, it's a bit of implementation detail that has leaked
into the method here.  It also prevents you from trying to clone from yourself :)